examples/next-liveness/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@aws-amplify/ui-next-example-liveness",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "clean": "rimraf node_modules .next",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@aws-amplify/ui-react": "5.2.0",
+    "@aws-amplify/ui-react-liveness": "^2.0.6",
+    "@aws-sdk/credential-providers": "^3.370.0",
+    "aws-amplify": "5",
+    "next": "^11.1.3",
+    "next-global-css": "^1.1.1",
+    "react": "17",
+    "react-copy-to-clipboard": "^5.1.0",
+    "react-dom": "17",
+    "react-icons": "^4.4.0",
+    "react-map-gl": "7.0.23",
+    "swr": "^2.0.0"
+  },
+  "devDependencies": {
+    "@types/node": "^15.12.4",
+    "@types/react": "17.0.47",
+    "eslint": "^8.44.0",
+    "eslint-config-next": "11.0.1",
+    "rimraf": "^3.0.2"
+  },
```

Comment:
Seems like a lot of these deps are kind of outdated?

Reply:
this was just copied from the example app, can try to update where necessary

Reply:
Think it would make the most sense to generate a new app entirely since our existing app is pretty far behind